README.md

```diff
@@ -4,10 +4,10 @@
 
 ## good stuff here too
 
-- 1
-- 2
-- 3
-- 4
+- 1. Test Web
+- 2. Project
+- 3. Deploy
+- 4. Repeat
```

Comment:
This doc is looking great so far. You might want to consider adding more context though.